src/components/Preview.css

```diff
@@ -59,6 +86,39 @@
   border-radius: 3px;
 }
 
+.Frame-directional-icon {
```

Comment:
`.Frame`-related -- can these not also go in the `Frame.css` file?

Reply:
Done